app/javascript/controllers/nested_clickable_controller.js

```diff
@@ -0,0 +1,23 @@
+import { Controller } from '@hotwired/stimulus'
+
+export default class extends Controller {
+
+  connect () {
+    this.element.addEventListener('mouseover', this.removeParentClickable)
+    this.element.addEventListener('mouseout', this.addParentClickable)
```

Comment:
Are these events working without calling `.bind(this)` on these? Maybe I'm missing something about some magic stimulus does for this...

Reply:
Yea, they're all working. I know there is a difference between these two ways of declaring a callback. Maybe this is why? 

addEventListener('mouseover', this.removeParentClickable)
addEventListener('mouseover', () => this.removeParentClickable())

Reply:
I did some reading about this and I am now thoroughly confused. I don't understand why it works. But noably, the click_toggle_controller fails to work as a stand-alone function unless I bind the this context.

Just to be thorough, I've convered them all to arrow functions just to ensure `this` is preserved.